openedx/openedx-platform
# test: check module store settings deprecations

## Description

There are two instances of the warning in openedx/public-engineering#103 according to the pytest warning report from the `unit-tests` workflow ([recent run](https://github.com/openedx/edx-platform/suites/13943274501/artifacts/776740271), under DeprecationWarning->Other). Both are fixed by this PR.

## Supporting information

Closes openedx/public-engineering#103

## Testing instructions

- Run `pytest xmodule/modulestore/tests/test_modulestore_settings.py` before and after this change to ensure deprecation warnings are no longer emitted
